backend/serializers/hacknight_serializer.py

```diff
@@ -0,0 +1,17 @@
+from marshmallow import Schema, fields
+
+
+class HacknightSchema(Schema):
+    class Meta:
+        fields = ('id', 'date', 'participants')
+        dump_only = ('id',)
+
+    participants = fields.Nested(
+        'ParticipantSchema',
+        only=('id', 'github'),
```

Comment:
I would prefer to have it consistent (if we're using `exclude` in `hacknights` would prefer it also here).

Also would believe, that if we're going to return a list of entities (`many=True`) list of fields could be limited, as participants list should not return hacknights list for every participant and don't see a lot of reason to return all participants in hacknights list.

Reply:
@stanislawK still valid (using exclude)